format.go

```diff
@@ -350,3 +350,80 @@ func Glob(input, match string) bool {
 	// Check suffix last.
 	return trailingGlob || strings.HasSuffix(input, parts[last])
 }
+
+const maxWordSplitLength = 30
+
+// splitAtWord is a text splitter that takes into consideration a few things:
```

Comment:
I would suggest adding a unit test, specifically for this function, to `format_test.go`.

Reply:
Yep, no tests yet (it's still listed in the TODO's of this PR).